index.js

```diff
@@ -0,0 +1,50 @@
+document.addEventListener('DOMContentLoaded', () => {
+  let booksCollection = [];
+  const bookList = document.querySelector('.booklist');
+  const titleInput = document.querySelector('#title');
+  const authorInput = document.querySelector('#author');
+  const addButton = document.querySelector('.add-button');
+
+  const displayBooks = () => {
+    bookList.innerHTML = '';
+    booksCollection.forEach((book) => {
+      const bookElement = document.createElement('div');
+      bookElement.innerHTML = `
+        <p>${book.title}</p>
+        <p>By</p>
+        <h2>${book.author}</h2>
+        <button class="remove-book" data-title="${book.title}">Remove</button>
+      `;
+      bookList.appendChild(bookElement);
+    });
+    const removeButtons = document.querySelectorAll('.remove-book');
+    removeButtons.forEach((button) => {
+      button.addEventListener('click', () => {
+        const title = button.getAttribute('data-title');
+        // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] I will implore you to remove this linter disabler and try to fix the issue, professionally, you should declare a function before calling it, in fact, that is the essence of the arrow function which came to relegate the function keyword. Kindly make this a part and parcel of you.